net-p2p/go-ipfs-bin/go-ipfs-bin-0.4.15.ebuild

```diff
@@ -25,5 +25,15 @@ QA_PREBUILT="/usr/bin/ipfs"
 src_install() {
 	dobin ipfs
 
+	systemd_dounit "${FILESDIR}/ipfs.service"
+	newinitd "${FILESDIR}/ipfs.init" ipfs
+
 	newbashcomp "${DISTDIR}/${P}.bash" "ipfs"
 }
+
+pkg_postinst() {
+	enewgroup ipfs
+	enewuser ipfs "" "" /var/lib/ipfs ipfs
+
+	su -s /bin/sh -c "ipfs init -e" ipfs
```

Comment:
better not to perfor such an actioon here but put a note about it

Reply:
Ok, I will change it. What's the reasoning behind it? If there are something that needs to be done so the program can work correctly, isn't best to do it as soon as possible?